### ISSUE TITLE
Cannot open file 'Dati.cart.out' 

### ISSUE THREAD

**Issue by lv4662** [davecats/channel-f90#2]
>./channel 
At line 493 of file dnsdata.f90 (unit = 100)
Fortran runtime error: Cannot open file 'Dati.cart.out': No such file or directory

There is no 'Dati.cart.out' file at the beginning of CALL read_restart_file(), PS: running with the defaut dns.in

**Comment by davecats** [comment]
Correct! Indeed, you need a velocity field to start with as initial condition. You could create one externally (for instance a field of small random complex numbers) for instance. 

If you look in my repository channel-f90-mpi you will find a more sophisticated version of this code, which can be also run in parallel. In this version, if the initial field does not exist it will automatically create one, superimposing a Poiseuille profile with some random disturbances. 

You can also modify the subroutine READ_RESTART_FILE for automatically creating an initial field by copying the one in the repository channel-f90-mpi. Be careful though! In the two codes, the order of the indices in the velocity vector changed!
